language_server.gemspec

```diff
@@ -31,6 +31,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.14"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest", "~> 5.0"
+  spec.add_development_dependency "guard"
+  spec.add_development_dependency "guard-minitest"
+  spec.add_development_dependency "minitest-color"
```

Comment:
[minor] I think adding minitest-color isn't related to this PR.
It's ok at this time but please split PR at next time 🙏